app/jobs/process_github_sponsor_update_job.rb

```diff
@@ -0,0 +1,12 @@
+class ProcessGithubSponsorUpdateJob < ApplicationJob
```

Comment:
Could this be a regular command with the special Mandate sidekiq magic?

Reply:
Sure!